pkg/sql/sem/tree/create.go

```diff
@@ -168,6 +168,58 @@ func (node *CreateIndex) Format(ctx *FmtCtx) {
 	}
 }
 
+// CreateTypeVariety represents a particular variety of user defined types.
+type CreateTypeVariety int
+
+//go:generate stringer -type=CreateTypeVariety
+const (
+	_ CreateTypeVariety = iota
+	// Enum represents an ENUM user defined type.
+	Enum
```

Comment:
wondering if there's any value to these being protobuf.

Reply:
Not that I know of (atm). This won't be serialized either